features/tags_and_wrangling/tag_set.feature

```diff
@@ -79,15 +79,190 @@ Feature: creating and editing tag sets
   Given I nominate 3 fandoms and 3 characters in the "Nominated Tags" tag set as "nominator"
     And I submit
   Then I should see "Your nominations were successfully submitted"
-    
+
+  Scenario: You should be able to edit your nominated tag sets, but cannot delete them once they've been reviewed
+  Given I am logged in as "tagsetter"
+    And I set up the nominated tag set "Mayfly" with 3 fandom noms and 3 character noms
+  Given I am logged in as "passerby"
+    And I go to the tagsets page
+    And I follow "Mayfly"
+    And I follow "Nominate"
+    And I nominate fandom "Floobry" and character "Barblah" in "Mayfly"
+  When I follow "Edit"
+  Then I should see "Floobry" within "li"
+  When I press "Submit"
+  Then I should see "Your nominations were successfully updated"
+  Given I am logged in as "tagsetter"
+    And I go to the tagsets page
+    And I follow "Mayfly"
+    And I follow "Review Nominations"
+  Then I should see "Floobry" within ".tagset"
+  When I check "fandom_approve_Floobry"
+    And I press "Submit"
+  Then I should see "Successfully added to set: Floobry"
+  Given I am logged in as "passerby"
+    And I go to the tagsets page
+    And I follow "Mayfly"
+    And I follow "My Nominations"
+    And I follow "Edit"
+  Then I should see "Floobry"
+    And I should see "This nomination has been approved!"
+    And I should not see "Floobry" within "li"
+
+  Scenario: Owner of a tag set can clear all nominations
+  Given I am logged in as "tagsetter"
+    And I set up the nominated tag set "Nominated Tags" with 3 fandom noms and 3 character noms
+  Given I nominate 3 fandoms and 3 characters in the "Nominated Tags" tag set as "nominator"
+    And I submit
+  Then I should see "Your nominations were successfully submitted"
+  Given I am logged in as "tagsetter"
+  When I review nominations for "Nominated Tags"
+    And I follow "Clear Nominations"
+    And I press "Yes, Clear Tag Set Nominations"
+  Then I should see "All nominations for this Tag Set have been cleared"
+
+  Scenario: Owner of a tag set with over 30 nominations sees a message that they can't all be displayed on one page
+  Given I am logged in as "tagsetter"
+    And I set up the nominated tag set "Nominated Tags" with 6 fandom noms and 6 character noms
```

Comment:
I would look at the method used in https://github.com/otwcode/otwarchive/blob/master/features/step_definitions/admin_steps.rb#L236